webpack.config.js

```diff
@@ -334,6 +335,8 @@ const settingsPage = {
 				}
 			},
 		} ),
+		// Ensure assets generated by `DependencyExtractionWebpackPlugin` are also ignored for styles entries.
```

Comment:
This should prevent the empty JS chunk and PHP asset file for the `wp-components` entry from being generated.

Reply:
Success:

![image](https://user-images.githubusercontent.com/134745/115311075-d8e8e100-a123-11eb-89c0-d35ed8310143.png)